examples/basic_usage/versions.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
```

Comment:
Should this be consistent with `versions.tf`?

Reply:
You mean with the same comments and such?

Reply:
Yes, I think you are right @mcdonnnj.  They probably should have the same comments and same restrictions on the AWS provider version to support `default_tags`.

Reply:
> If we're all on board here, I will add that change into `skeleton-tf-module/examples/basic_usage/versions.tf` so that it filters down.

@jsf9k and @mcdonnnj - what do you say?

Reply:
I'm fine with either of wholesale copying everything or at mininum mirroring the version restriction for the `aws` provider for `default_tags` support.

Reply:
I was planning to just copy the whole file.  I'll put together the PR now.  If @jsf9k has any issues, we can sort them out in the PR.

Reply:
See https://github.com/cisagov/skeleton-tf-module/pull/58 for this change.

Reply:
@mcdonnnj - Should be good to go now.